internal/test/wmcb/wmcb_test.go

```diff
@@ -202,11 +202,11 @@ func (vm *wmcbVM) initializeTestBootstrapperFiles() error {
 		return fmt.Errorf("unable to copy kubelet.exe to %s", winTemp)
 	}
 
-	// The 0.35.0 maps to ignition spec v2. This should be modified when we switch to v3
-	ignitionUserAgentSpec := "Ignition/0.35.0"
+	// Ignition v2.3.0 maps to Ignition config spec v3.1.0.
+	ignitionAcceptHeaderSpec := "application/vnd.coreos.ignition+json`;version=3.1.0"
```

Comment:
If possible, please keep the 2.x config as well to exercise the code path. It's fine to keep if you think this is going to be a throw away work

Reply:
Agree, we should still test it as long as the functionality is there. I'll add back a spec v2 test in the second commit.